crates/stdarch-gen-loongarch/src/main.rs

```diff
@@ -281,27 +402,27 @@ fn gen_bind_body(
             let fn_output = if out_t.to_lowercase() == "void" {
                 String::new()
             } else {
-                format!(" -> {}", type_to_rst(out_t, is_store))
+                format!(" -> {}", type_to_rst(out_t, is_store, true))
```

Comment:
would using a custom enum instead of a `bool` be nicer here? just reading this it's hard to know what `true` means.

Reply:
Good idea. Thank you!